sys/include/net/gnrc/pkt.h

```diff
@@ -109,18 +109,18 @@ typedef struct gnrc_pktsnip {
     /* the first three fields *MUST* match iolist_t! */
     struct gnrc_pktsnip *next;      /**< next snip in the packet */
     void *data;                     /**< pointer to the data of the snip */
-    size_t size;                    /**< the length of the snip in byte */
+    uint16_t size;                  /**< the length of the snip in byte */
```

Comment:
Proposal for compromise: How about a typedef `gnrc_pkt_size_t`, which if need be can be increased in size with a simple MACRO switch?

Reply:
Ok